modules/server/src/mapping/masking.ts

```diff
@@ -1,10 +1,11 @@
 import { ENV_CONFIG } from '@/config';
-import { Aggregation } from './types';
+import { Aggregations, AllAggregationsMap } from './resolveAggregations';
```

Comment:
are these types, classes? think if they're types, we could use adding the explicit import (e.g. `import type { ... }`) for absolute clarity on what these are